nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (request, response) => {
+  try {
+    console.log(meals);
+    console.log("in /api/meals");
+
+    const maxPrice = request.query.maxPrice;
+    const titles = request.query.title;
+    const createdAfter = request.query.createdAfter;
+    const limit = request.query.limit;
+    if (maxPrice !== undefined && maxPrice !== "") {
+      console.log("maxPrice");
+      const filteredMeals = meals.filter((meal) => meal.price < maxPrice);
+      response.send(filteredMeals);
+    } else if (titles !== undefined && titles !== "") {
+      console.log("titles");
+      const filteredTitle = meals.filter((meal) => meal.title.includes(titles));
+      response.send(filteredTitle);
+    } else if (createdAfter !== undefined && createdAfter !== "") {
+      console.log("createdAfter");
+      const filteredDate = meals.filter(
+        (meal) => meal.createdAt > createdAfter
```

Comment:
similar as the maxPrice, could you try converting the createdAfter variabe to a Date variable? This is currently a string variable